chartist-plugin-legend.js

```diff
@@ -36,6 +42,18 @@
             return a - b;
         }
 
+        // Parse input position string to enum
```

Comment:
Not essential to the functionality, no, if you really want me to, i can change it.
Although it allows way easier intergration of possible additional positions in the future, and also the code is much cleaner IMHO; also i regard it as good practice :)